### PR TITLE
Bug 1990190: Make Manila CSI job voting where it's useful

### PR DESCRIPTION
**Pull request by EmilienM**
The job should be stable now and voting where it's important:
* cloud-provider-openstack
* csi-driver-manila-operator
* csi-driver-nfs

### CODE REVIEW

File: ci-operator/step-registry/openshift/e2e/openstack/csi/manila/openshift-e2e-openstack-csi-manila-workflow.yaml [hunk 1]
```diff
@@ -14,6 +14,12 @@ workflow:
       CONFIG_TYPE: "minimal"
       BASE_DOMAIN: shiftstack.devcluster.openshift.com
       TEST_CSI_DRIVER_MANIFEST: manifest-openstack-manila.yaml
+      # We have negative tests which provoke errors and make sure
+      # we have them, so we don't want to run this test which
+      # always fails since we have these errors on purpose.
+      # https://bugzilla.redhat.com/show_bug.cgi?id=1990190
+      TEST_SKIPS: >-
+        events should not repeat pathologically
```
Comment (jsafrane) [inline ci-operator/step-registry/openshift/e2e/openstack/csi/manila/openshift-e2e-openstack-csi-manila-workflow.yaml:22]: Did you try to skip the offending test instead? Or would it still produce the events?

**Comment by deads2k** [comment]
Instead of overriding pass or fail from here, add the exception where the test is defined here: https://github.com/openshift/origin/blob/master/pkg/synthetictests/duplicated_events.go#L51-L53

It's fine to do so and you can see several examples of expected repeated events that we skip today.

File: ci-operator/step-registry/openshift/e2e/openstack/csi/manila/openshift-e2e-openstack-csi-manila-workflow.yaml (hunk 1 above)
Comment (EmilienM) [reply ci-operator/step-registry/openshift/e2e/openstack/csi/manila/openshift-e2e-openstack-csi-manila-workflow.yaml:22]: I'm still investigating this issue, I skipped the test locally and I still see this "events should not repeat" test failing...

**Comment by deads2k** [comment]
/hold

for the preferred resolution here https://github.com/openshift/release/pull/20932#issuecomment-894234963

File: ci-operator/step-registry/openshift/e2e/openstack/csi/manila/openshift-e2e-openstack-csi-manila-workflow.yaml (hunk 1 above)
Comment (EmilienM) [reply ci-operator/step-registry/openshift/e2e/openstack/csi/manila/openshift-e2e-openstack-csi-manila-workflow.yaml:22]: BTW we can't skip this test, it seems. 

```
Error: no tests were found matching the TEST_SKIPS regex:
events should not repeat pathologically
```

**Comment by pierreprinetti** [comment]
/retitle fix OpenStack Manila CSI job and make it voting

**Comment by pierreprinetti** [comment]
/retitle Bug 1990190: fix OpenStack Manila CSI job and make it voting

**Comment by iamemilio** [comment]
This might have to wait for: https://github.com/openshift/kubernetes/pull/1011

**Comment by iamemilio** [comment]
:cry: why is it still broken

**Comment by mandre** [comment]
> cry why is it still broken

You'll need to revendor openshift/kubernetes into openshift/origin for it to take effect. See https://github.com/openshift/origin/pull/26491 as an example.

**Pull request merged**